setup.cfg

```diff
@@ -0,0 +1,41 @@
+[wheel]
+universal=1
+
+[metadata]
+license_file = apache-2.0.LICENSE
+name = Fetchcode
+author = nexB. Inc. and others
+author_email = info@aboutcode.org
+description = It is a library to reliably fetch code via HTTP, FTP and version control systems.
+long_description = file:README.rst
+url = https://github.com/nexB/fetchcode
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
```

Comment:
I forget, did we start this repository with python3?

Reply:
Yes we started this repo with python3